node/src/accountant/payment_adjuster/service_fee_adjuster.rs

```diff
@@ -22,89 +17,75 @@ use crate::accountant::payment_adjuster::logging_and_diagnostics::diagnostics::o
 pub trait ServiceFeeAdjuster {
     fn perform_adjustment_by_service_fee(
         &self,
-        weighted_accounts: Vec<WeightedPayable>,
+        weighed_accounts: Vec<WeighedPayable>,
         disqualification_arbiter: &DisqualificationArbiter,
-        unallocated_cw_service_fee_balance_minor: u128,
+        remaining_cw_service_fee_balance_minor: u128,
         logger: &Logger,
     ) -> AdjustmentIterationResult;
 }
 
-pub struct ServiceFeeAdjusterReal {
-    adjustment_computer: AdjustmentComputer,
-}
+#[derive(Default)]
+pub struct ServiceFeeAdjusterReal {}
 
 impl ServiceFeeAdjuster for ServiceFeeAdjusterReal {
     fn perform_adjustment_by_service_fee(
         &self,
-        weighted_accounts: Vec<WeightedPayable>,
+        weighed_accounts: Vec<WeighedPayable>,
         disqualification_arbiter: &DisqualificationArbiter,
         cw_service_fee_balance_minor: u128,
         logger: &Logger,
     ) -> AdjustmentIterationResult {
-        let unconfirmed_adjustments = self
-            .adjustment_computer
-            .compute_unconfirmed_adjustments(weighted_accounts, cw_service_fee_balance_minor);
+        let unconfirmed_adjustments =
+            compute_unconfirmed_adjustments(weighed_accounts, cw_service_fee_balance_minor);
 
-        let checked_accounts = Self::handle_winning_accounts(unconfirmed_adjustments);
+        let checked_accounts = Self::try_confirm_some_accounts(unconfirmed_adjustments);
 
         match checked_accounts {
-            Either::Left(no_thriving_competitors) => Self::disqualify_single_account(
+            Either::Left(no_accounts_above_disq_limit) => Self::disqualify_single_account(
```

Comment:
A nice suggestion from you is to rename `no_accounts_above_disq_limit` to `only_accounts_before_disq_limit`.